src/main/lrsql/admin/routes.clj

```diff
@@ -30,68 +31,123 @@
     ;; Create new account
     ["/admin/account/create" :post (conj common-interceptors
                                          ai/validate-params
-                                         (ji/validate-jwt jwt-secret jwt-leeway)
+                                         (ji/validate-jwt
+                                          jwt-secret jwt-leeway
+                                          :no-val? no-val?
+                                          :no-val-uname no-val-uname
+                                          :no-val-issuer no-val-issuer
+                                          :no-val-role-key no-val-role-key
+                                          :no-val-role no-val-role)
```

Comment:
Now that lrsql is on Clojure 1.11.1, you can pass in these extra kwargs as an opt map, which will clear up all this kwarg clutter. That said, since all these args are being passed together, maybe you should make these args to `validate-jwt` an opt map in the first place (or maybe make the opt map itself a kwarg, like `:no-val-opts`).